openwisp_monitoring/monitoring/static/monitoring/js/chart.js

```diff
@@ -193,6 +308,12 @@
             charts.push(options);
         }
         charts = sortByTraceOrder(data.trace_order, charts, '_key');
+
+        if(unit == 'adaptive_bytes') {
```

Comment:
```suggestion
        if (unit == 'adaptive_bytes') {
```

Let's leave a space before the rounded parenthesis in code blocks like this one or in `for`, please apply the same changes to other lines as well.

@codesankalp @pandafy how can we enforce this in the linter?

Reply:
I think we should start using `eslint` in `openwisp-utils`, It is much more configurable than any other linter package.
Also, Django is using this as a linter in JavaScript code: https://github.com/django/django/blob/main/.eslintrc